plugin/user_plugin_server.py

```
@@ -0,0 +1,558 @@
+                        # 有些装饰器可能将 meta 绑定到函数的 __wrapped__（例如 functools.wraps 情况），尝试获取
+                        wrapped = getattr(member, "__wrapped__", None)
+                        if wrapped is not None:
+                            event_meta = getattr(wrapped, EVENT_META_ATTR, None)
+                    if event_meta is None:
+                        continue
+                    # 仅关注 plugin_entry 类型
+                    try:
+                        if getattr(event_meta, "event_type", None) != "plugin_entry":
+                            continue
+                    except Exception:
+                        continue
```

Ruff (0.14.7)

`try`-`except`-`continue` detected, consider logging the exception

(S112)